### PR TITLE
Added scroll to top button

### PR DESCRIPTION
**Pull request by anishagg17**
Hnet com-image-2

### CODE REVIEW

**Comment by mariobehling** [comment]
Please also provide a link for testing where you deployed the website on gh-pages.

**Comment by anishagg17** [comment]
preview at :  https://anishagg17.github.io/labs.fossasia.org/

**Comment by mariobehling** [comment]
Often back to top buttons fade in when the user moves the page and fade out when the user stops interacting with the page. Could you try to make this work in the same way, please?

**Comment by anishagg17** [comment]
@mariobehling i have done changes as they were supposed.
Please have a look  https://anishagg17.github.io/labs.fossasia.org/

**Comment by mariobehling** [comment]
Please check the fade-in delay. This delay is too long.

**Comment by anishagg17** [comment]
Please have a look https://anishagg17.github.io/labs.fossasia.org/

**Comment by mariobehling** [comment]
Please also make it work nicely on mobile.

Screenshot_20191214_102901_com android chrome

**Comment by anishagg17** [comment]
It's working fine on my device can you please explain what problem are you facing?

**Comment by mariobehling** [comment]
The screen should only be scrollable up and down. Your changes result in a horizontal scroll bar.

**Comment by anishagg17** [comment]
I have updated please have a 
view

**Comment by mariobehling** [comment]
Oh that problem was already there before. I will open an issue. Thanks.

File: js/scripts.js [hunk 1]
```diff
@@ -338,4 +338,5 @@ $(window).load(function() {
     }, 10);
 
 
-}); 
\ No newline at end of file
+}); 
+
```
Comment (mariobehling) [inline js/scripts.js]: Please don't change script.js

File: index.html [hunk 2]
```diff
@@ -686,6 +697,8 @@ <h1>How Can I Participate in FOSSASIA projects?</h1>
 		window.setTimeout(function() {
 		    offsetAnchor();
 		}, 1);
+
+
```
Comment (mariobehling) [inline index.html]: Please don't make these kind of changes.

File: js/scripts.js (hunk 1 above)
Comment (anishagg17) [reply js/scripts.js]: okay.

File: index.html [hunk 3]
```diff
@@ -371,6 +372,12 @@ <h3>Incubation Lab</h3>
 	<section class="duplicatable-content">
 					<div class="container">				
 						<div class="row">
+
+							<div class="col-sm-12 text-center" id="gsoc-proj">
+								<h1>Project Ideas for Coding Programs and Google Summer of Code</h1>
+                                <div id="projects_holder" style="background-color: #eee;">
+							<div class="col-sm-14" id="gsoc-proj">
+								<h1></h1>
```
Comment (mariobehling) [inline index.html]: Please don't make these changes here. They are not related to the issue.

**Comment by mariobehling** [comment]
Please only make changes in this PR, which are related to the issue. Thanks.

**Comment by anishagg17** [comment]
I have reverted all changes to only those which are related to the pr

**Comment by mariobehling** [comment]
Please only change files, if needed, e.g. script.js still shows up as edited. This is not needed. There is also a .DS file for no reason.

**Comment by anishagg17** [comment]
Sir, i tried many times to remove that change from script.js file but i don't know why isn't it reverting

**Comment by anishagg17** [comment]
Sir ,I haven't changed any thing in scripts.js
I have tried a lot to revert it but I couldn't

**Comment by anishagg17** [comment]
Sir actually that new-line in scripts.js is added by github not by me.
I had touched that file and then reverted to original one but as soon as I commit I notice that is added by github.
It may be because that file was made years ago and git introduced various changes

**Comment by anishagg17** [comment]
Sir actually that new line in scripts.js is added by github
You may refer to this article:
https://stackoverflow.com/questions/51049503/github-markdown-that-respects-newlines

It occurs because scripts.js file was untouched after git updated .Since now I made a change and reverted it Git detects that file and inserted new line at the end of file

**Comment by anishagg17** [comment]
You can observe it by changing any file which doesn't end with a new line

**Pull request merged**